VersionInfo.rc.in

```diff
@@ -0,0 +1,48 @@
+#include <Windows.h>
+
+/* UTF-8 */
+#pragma code_page 65001
+
+#cmakedefine AC_ADDON_IS_RELEASE
```

Comment:
This define will come from the add-on's `CMakeLists.txt` file later. This is not an indication whether the binary is built using release optimizations and release runtime, but that the built binary is meant for public distribution. Is this name adequate to signal this meaning? Will just having this mentioned in the cache variable's help text enough?

Reply:
I would use a different name than RELEASE, exactly for what you mentioned: not to confuse it with the build type.
E.g., AC_ADDON_FOR_DISTRIBUTION